### DISCUSSION TITLE
Vulnerability issue in dropwizard-testing dependency : 2.1.1

### DISCUSSION THREAD

**Discussion by viksin19** [dropwizard/dropwizard#5869] [General]
After running the OWASP Dependency checker against the dropwizard, one vulnerability is raised against dropwizard-testing dependency which is brought in:
grizzly-http-client-1.16.jar: CVE-2021-43138

![image](https://user-images.githubusercontent.com/52523831/189886949-b3443993-2ab1-495c-8f52-fe6c22c793ac.png)

**Comment by joschi** [comment]
This is a false positive and https://github.com/advisories/GHSA-fwr7-v2mv-hh25 is about an npm package (JavaScript). Please at least triage these things before you blindly post them here. 🙄
